<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WNewRelease</class>
 <widget class="QWidget" name="WNewRelease">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Clean">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>333</y>
     <width>215</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Clean
{
    background-color: #a3f48a;
    color: black;
    font-size: 12px;
}</string>
   </property>
   <property name="text">
    <string>Limpar</string>
   </property>
  </widget>
  <widget class="QLabel" name="Description">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Descricao:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt;</string>
   </property>
   <property name="text">
    <string>Realizar Novo Lancamento</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Confirm">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>333</y>
     <width>215</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Confirm
{
    background-color: #a3f48a;
    color: black;
    font-size: 12px;
}</string>
   </property>
   <property name="text">
    <string>Confirmar</string>
   </property>
  </widget>
  <widget class="QLabel" name="Description_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lancamento que registra as operacoes feitas sobre a tua conta.</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ValueBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#ValueBox
{
	color: black;
	border: none;
}</string>
   </property>
   <property name="maximum">
    <double>99999999999999995287335453651211007997446182781858083179085387749785952239205787068995699003416510776387310061494932420984963311567802202010637287727642443776.000000000000000</double>
   </property>
  </widget>
  <widget class="QRadioButton" name="In">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>197</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrada</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Out">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>197</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Saida</string>
   </property>
  </widget>
  <widget class="QLabel" name="Operation">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>197</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Operacao:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Value">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="DescRelease">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Name
{
	border: none;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="Erro">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Erro
{
	color: red;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="DateEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>197</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#DateEdit
{
	border: none;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="Date">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>197</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Data:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Accounts">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>94</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>Conta</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="ReleaseType">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>128</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true">Tipo de Lancamento</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="PaymentType">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>163</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Tipo de Pagamento</string>
    </property>
    <property name="icon">
     <iconset theme="color:red;">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
